src/concrete/ml/torch/hybrid_model.py

```diff
@@ -476,17 +460,16 @@ def forward(self, x: torch.Tensor, fhe: str = "disable") -> torch.Tensor:
 
                 # Loading keys from a file could be done here, and the
                 # keys could be passed as arguments to the Executor
-                executor = GLWELinearLayerExecutor()
-
+                self.executor = GLWELinearLayerExecutor()
                 if fhe_mode != HybridFHEMode.DISABLE:
-                    executor.keygen()
+                    self.executor.keygen()
```

Comment:
this is doing for every hybridmodel.forward no ? 

I wanted to avoid that: one keygen should be done a single time and the key re-used

Moreover, we should think about that, at some point,we need the option to load keys from a key store

Reply:
That's not how the hybrid model with concrete works. We can't assume that all circuit can reuse the same key. But yes I get your point.

Reply:
I updated. We should only have the initialization and keygen once